From 737536716807520760013d910b18d49175c5d7f4 Mon Sep 17 00:00:00 2001
From: Sachin Prabhu <sprabhu@redhat.com>
Date: Mon, 28 Jul 2014 12:16:27 -0400
Subject: [fs] cifs: Removed duplicated (and unneeded) goto

Message-id: <1406549822-10376-35-git-send-email-sprabhu@redhat.com>
Patchwork-id: 86789
O-Subject: [RHEL7.1 PATCH 34/69] [bz1082049]Removed duplicated (and unneeded) goto
Bugzilla: 1082049
RH-Acked-by: J. Bruce Fields <bfields@redhat.com>
RH-Acked-by: Steve Dickson <SteveD@redhat.com>

From: Steve French <smfrench@gmail.com>

Remove an unneeded goto (and also was duplicated goto target name).

Signed-off-by: Steve French <smfrench@gmail.com>
(cherry picked from commit 7d3fb24bce87a240ee5a5f99cdd72b1f336d5c3b)
Signed-off-by: Sachin Prabhu <sprabhu@redhat.com>
---

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/fs/cifs/smb2pdu.c b/fs/cifs/smb2pdu.c
index 62be20f..d24f014 100644
--- a/fs/cifs/smb2pdu.c
+++ b/fs/cifs/smb2pdu.c
@@ -2162,11 +2162,9 @@ send_set_info(const unsigned int xid, struct cifs_tcon *tcon,
  rc = SendReceive2(xid, ses, iov, num, &resp_buftype, 0);
  rsp = (struct smb2_set_info_rsp *)iov[0].iov_base;
 
- if (rc != 0) {
+ if (rc != 0)
   cifs_stats_fail_inc(tcon, SMB2_SET_INFO_HE);
-  goto out;
- }
-out:
+
  free_rsp_buf(resp_buftype, rsp);
  kfree(iov);
  return rc;
-- 
1.7.1